### PR TITLE
Wali/pipeline token issue fix

### PR DESCRIPTION
**Pull request by waliMSFT**
- [ ] The purpose of this PR is explained in this message or in an issue. If an issue please include a reference as \#\<issue\_number\>.
- [ ] Tests are included and/or updated for code changes.
- [ ] Proper license headers are included in each file.

### CODE REVIEW

File: images/build/installDotNetCore.sh [hunk 1]
```diff
@@ -24,9 +24,9 @@ if [ "$sdkStorageAccountUrl" == "$PRIVATE_STAGING_SDK_STORAGE_BASE_URL" ]; then
     set -x
     
     if [ $isSasTokenEmpty -eq 0 ]; then
-      echo "sasToken cannot be empty for $sdkStorageAccountUrl."
+        echo "sasToken cannot be empty for $sdkStorageAccountUrl."
     else
-        echo "sasToken is valid for $sdkStorageAccountUrl."
+        echo "sasToken is empty for $sdkStorageAccountUrl."
```
Comment (cormacpayne) [inline images/build/installDotNetCore.sh:29]: Is this correct? `isSasTokenEmpty` is set to 0 on line 22 if the value of `sasToken` provided is empty, so the above if-statement is correct and this `else` statement should only be hit if `sasToken` has some value.

File: images/installPlatform.sh [hunk 2]
```diff
@@ -61,7 +61,7 @@ if [ "$sdkStorageAccountUrl" == "$PRIVATE_STAGING_SDK_STORAGE_BASE_URL" ]; then
     if [ $isSasTokenEmpty -eq 0 ]; then
       echo "sasToken cannot be empty for $sdkStorageAccountUrl."
     else
-        echo "sasToken is valid for $sdkStorageAccountUrl."
+      echo "sasToken is empty for $sdkStorageAccountUrl."
```
Comment (cormacpayne) [inline images/installPlatform.sh:64]: Same comment here.

File: vsts/pipelines/baseImages/dotnetcore.yml [hunk 3]
```diff
@@ -1,8 +1,13 @@
...
-    Packaging.EnableSBOMSigning: true
+- group: Oryx
+- name: ascName
+  value: oryx-automation-service-principal
+- name: acrName
+  value: oryxdevmcr.azurecr.io
+- name: skipComponentGovernanceDetection
+  value: true
+- name: Packaging.EnableSBOMSigning
+  value: true
```
Comment (cormacpayne) [inline vsts/pipelines/baseImages/dotnetcore.yml:10]: nit: do we need to change the format of all of these existing variables to add the Oryx group? Would we not be able to do something like the following:

```yaml
variables:
  group: Oryx
  ascName: oryx-automation-service-principal
  acrName: oryxdevmcr.azurecr.io
  skipComponentGovernanceDetection: true
  Packaging.EnableSBOMSigning: true
```

File: images/build/installDotNetCore.sh [hunk 4]
```diff
@@ -24,9 +24,9 @@ if [ "$sdkStorageAccountUrl" == "$PRIVATE_STAGING_SDK_STORAGE_BASE_URL" ]; then
     set -x
     
     if [ $isSasTokenEmpty -eq 0 ]; then
-      echo "sasToken cannot be empty for $sdkStorageAccountUrl."
+        echo "sasToken cannot be empty for $sdkStorageAccountUrl."
```
Comment (snehapar9) [inline images/build/installDotNetCore.sh:27]: Is the space here before `echo` intended?

**Comment by william-msft** [comment]
> @waliMSFT Could you please confirm that we don't need to be checked in before the PR from @william-msft for .NET 8 preview? I want to make sure that we don't delay that last one.

@daniv-msft we'll need a fix for the SAS token issue in Oryx-CI pipeline: https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=8305442&view=logs&j=f45c9538-ea56-56fc-09d6-585ee0335e0c&t=4c7d1d52-4268-5964-8ee9-edcf23ad5c3c&l=2613

**Pull request closed without merging**